### PR TITLE
Update test frequency for ocp 4.12 z-stream upgrade

### PR DESCRIPTION
**Pull request by JianLi-RH**
/cc @jianlinliu @jiajliu @shellyyang1989

### CODE REVIEW

**Comment by JianLi-RH** [comment]
Except below jobs, all other job's frequency have extended to f60:
```
openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml
[jianl@jianl-thinkpadt14gen4 release]$ grep -e "-f28" $file
- as: aws-ipi-ovn-ipsec-f28
- as: azure-ipi-workers-rhel8-f28
- as: baremetalds-ipi-ovn-ipv4-fips-f28
- as: gcp-upi-private-xpn-ingress-glb-f28
- as: vsphere-ipi-proxy-workers-rhel8-f28
[jianl@jianl-thinkpadt14gen4 release]$ 
```

```
openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-stable-4.11.yaml
[jianl@jianl-thinkpadt14gen4 release]$ grep -e "-f28" $file
- as: nutanix-ipi-fips-f28
[jianl@jianl-thinkpadt14gen4 release]$ 
```

```
openshift-openshift-tests-private-release-4.12__arm64-nightly-4.12-upgrade-from-stable-4.11.yaml
[jianl@jianl-thinkpadt14gen4 release]$ grep -e "-f28" $file
- as: aws-ipi-proxy-sts-f28
[jianl@jianl-thinkpadt14gen4 release]$ 
```

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml [hunk 1]
```diff
@@ -60,8 +60,8 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: aws-c2s-ipi-disc-priv-fips-f28
-  cron: 45 15 2 * *
+- as: aws-c2s-ipi-disc-priv-fips-f60
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml:63]: This is Y-stream upgrade ci jobs, not z-stream upgrade ci, right?

Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-4.12-upgrade-from-stable-4.11.yaml:63]: faint...time always wasted by such stupid issue

**Comment by JianLi-RH** [comment]
open

**Pull request closed without merging**